server/src/test/java/org/elasticsearch/action/admin/indices/delete/DeleteIndexBlocksIT.java

```diff
@@ -38,6 +39,7 @@ public void testDeleteIndexWithBlocks() {
             assertBlocked(client().admin().indices().prepareDelete("test"), MetaData.CLUSTER_READ_ONLY_BLOCK);
         } finally {
             setClusterReadOnly(false);
+            assertAcked(client().admin().indices().prepareDelete("test").get());
```

Comment:
You don't have to add this manual delete.

Reply:
I was testing something. removing it now.